--- inputrc_old.aug	2023-01-16 13:20:41.772611288 +0100
+++ inputrc_new.aug	2023-01-16 13:19:39.764443515 +0100
@@ -40,18 +40,23 @@
                . key (Rx.word - "entry") . Sep.space
                . store Rx.word . Util.eol ]
 
+(* View: include
+     An include directive *)
+let include = [ Util.del_str "$include" . label "@include"
+              . Sep.space . store Rx.space_in . Util.eol ]
+
 (* View: condition
      An "if" declaration, recursive *)
 let rec condition = [ Util.del_str "$if" . label "@if"
                     . Sep.space . store Rx.space_in . Util.eol
-                    . (Util.empty | Util.comment | condition | variable | entry)*
+                    . (Util.empty | Util.comment | condition | include | variable | entry)*
                     . [ Util.del_str "$else" . label "@else" . Util.eol
-                      . (Util.empty | Util.comment | condition | variable | entry)* ] ?
+                      . (Util.empty | Util.comment | condition | include | variable | entry)* ] ?
                     . Util.del_str "$endif" . Util.eol ]
 
 (* View: lns
      The inputrc lens *)
-let lns = (Util.empty | Util.comment | condition | variable | entry)*
+let lns = (Util.empty | Util.comment | condition | include | variable | entry)*
 
 (* Variable: filter *)
 let filter = incl "/etc/inputrc"
