--- systemd_orig.aug	2022-04-07 15:27:18.219513977 +0200
+++ systemd_new.aug	2022-04-08 14:38:55.538494885 +0200
@@ -124,7 +124,7 @@
   in let arg  = [ seq "args" . sto_value ]
   in let args = [ counter "args" . label "arguments"
                 . (value_sep . arg)+ ]
-  in entry_fn entry_command_kw ( entry_command_flags . cmd . args? )?
+  in entry_fn entry_command_kw ( entry_command_flags . Util.del_opt_ws "" . cmd . args? )?
 
 (* View: entry_env
    Entry that takes a space separated set of ENV=value key/value pairs *)
@@ -132,12 +132,11 @@
      let envkv (env_val:lens) = key env_key . Util.del_str "=" . env_val
      (* bare has no spaces, and is optionally quoted *)
   in let bare = Quote.do_quote_opt (envkv (store /[^#'" \t\n]*[^#'" \t\n\\]/)?)
-  in let bare_dqval = envkv (store /"[^#"\t\n]*[^#"\t\n\\]"/)
-  in let bare_sqval = envkv (store /'[^#'\t\n]*[^#'\t\n\\]'/)
-     (* quoted has at least one space, and must be quoted *)
+  in let bare_dqval = envkv (store /"[^#"\t\n]*"/)
+  in let bare_sqval = envkv (store /'[^#'\t\n]*'/)
   in let quoted = Quote.do_quote (envkv (store /[^#"'\n]*[ \t]+[^#"'\n]*/))
   in let envkv_quoted = [ bare ] | [ bare_dqval ] | [ bare_sqval ] | [ quoted ]
-  in entry_fn entry_env_kw ( Build.opt_list envkv_quoted value_sep )
+  in entry_fn entry_env_kw ( Util.del_opt_ws "" . ( Build.opt_list envkv_quoted value_sep ))
 
 
 (************************************************************************
